<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Booking List</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1e1e1e;
                color: #b8b8b8;
                font-family: Arial;
            </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Liste des réservations pour la date du jour :</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateToday">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelLegend">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Légende :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTocome">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Parties à venir</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStarted">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Parties en cours</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSoon">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Parties temps restant &lt; 5 minutes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFinished">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Parties terminées</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="bookingList">
     <property name="styleSheet">
      <string notr="true">background-color: #1e1e1e;
                            color: #b8b8b8;
                            font-family: Arial;
                            selection-background-color: #3A3238;
                            selection-color: #b8b8b8;
                        </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
